src/scripts/compute_british_synonyms.py

```diff
@@ -46,28 +46,45 @@ def translate_be(df, dict):
 
 df_labels['be']=translate_be(df_labels,df_dict)
 df_synonyms['be']=translate_be(df_synonyms,df_dict)
+df_synonyms[['term','be','synonym_type']].dropna(inplace=True)
 
 df_be_syns = df_labels[['term','be']].dropna()
-df_be_syns.append(df_synonyms[['term','be']].dropna(),ignore_index = True)
-
-print(df_be_syns.head())
-
-df_rem = df_labels[['term','be']]
-df_rem = df_rem.append(df_synonyms[['term','be']])
-df_rem.drop_duplicates(inplace=True)
-
-df_be_syns.drop_duplicates(inplace=True)
+df_be_syns['synonym_type'] = "http://www.geneontology.org/formats/oboInOwl#hasExactSynonym"
+print(len(df_be_syns))
+df_be_syns = pd.concat([df_be_syns[['term','be','synonym_type']].copy(),df_synonyms])
+print(len(df_be_syns))
 
-df_be_syns=df_be_syns[~df_be_syns['be'].isin(df_synonyms['annotation'].tolist())]
-df_be_syns=df_be_syns[~df_be_syns['be'].isin(df_labels['annotation'].tolist())]
-df_be_syns['Type']="http://purl.obolibrary.org/obo/hp#uk_spelling"
-df_be_syns.loc[-1] = ['ID', 'A http://www.geneontology.org/formats/oboInOwl#hasExactSynonym','>AI http://www.geneontology.org/formats/oboInOwl#hasSynonymType']  # adding a row
-df_be_syns.index = df_be_syns.index + 1  # shifting index
-df_be_syns.sort_index(inplace=True) 
+for syn_type in df_synonyms['synonym_type'].unique():
+	print(syn_type)
+	print(df_be_syns.head())
+	print("A")
```

Comment:
Is printing 'A" necessary? Good to minimize shell noise!

Reply:
Absolutely NOT ahhh! Good catch (removed now). But none of this will impact regular QC in any case - this is just run manually from time to time!